draft-ietf-ppm-dap.md

```diff
@@ -827,8 +827,12 @@ The `nonce` and `extensions` fields of each ReportShare match that in the Report
 uploaded by the client. The `encrypted_input_share` field is the `HpkeCiphertext`
 whose index in `Report.encrypted_input_shares` is equal to the index of the aggregator
 in the task's `aggregator_endpoints` to which the AggregateInitializeReq is being sent.
-The `agg_param` field is an opaque, VDAF-specific aggregation parameter. The
-`job_id` parameter contains the leader's chosen AggregationJobID.
+The `agg_param` field is an opaque, VDAF-specific aggregation parameter which is constant per collection flow.
+Note that this is the same value as in `CollectReq` and `AggregateShareReq`.
+Bandwith can be optimized by referencing the value from `AggregateInitializeReq` in `AggregateShareReq`.
```

Comment:
While tempting, I don't think this is good advice. The leader doesn't necessarily know that helper will know how to interpret `agg_param` in an `AggregateShareReq` as a reference. In any case, it seems like premature optimization to me, so I'd suggest removing it.

Reply:
tbh I am not quite sure why we are sending it twice. Maybe elaborating on that here would be more useful.

Reply:
The agg param is needed here because this the first time the Helper learns its value. The AggregateShareReq message isn't sent until later, once the batch is ready to be collected.

If we wanted to save bandwidth, we could consider replacing AggregateShareReq.agg_param with something like SHA-256(agg_param). However I agree that this is premature and can wait until someone has implemented Poplar1.